### ISSUE TITLE
Datapack error after reloading world, persists until game is rebooted

### ISSUE THREAD

**Issue by pajicadvance** [sisby-folk/tinkerers-smithing#16]
Mod version 2.3.0, Quilt loader

Steps to reproduce:
1. Create a new world or open an existing world
2. Exit the world
3. Open the same world
4. Datapack error:
![image](https://github.com/sisby-folk/tinkerers-smithing/assets/32738446/96805510-3420-45f1-94bb-41dc35cfea07)

- The error persists, and only goes away after exiting the game and launching it again.
- Trying to create a new world crashes the game.

Relevant log output:
```
[14:45:06] [Render thread/WARN]: Failed to load level data or datapacks, can't proceed with server load
java.util.concurrent.ExecutionException: java.lang.NullPointerException: Cannot read field "TOOL_MATERIALS" because the return value of "folk.sisby.tinkerers_smithing.TinkerersSmithing.getLoaderInstance()" is null
...
Caused by: java.lang.NullPointerException: Cannot read field "TOOL_MATERIALS" because the return value of "folk.sisby.tinkerers_smithing.TinkerersSmithing.getLoaderInstance()" is null
	at folk.sisby.tinkerers_smithing.data.SmithingToolMaterialLoader.getOutputMap(SmithingToolMaterialLoader.java:46) ~[transformed-mod-tinkerers-smithing.i0:0/:?]
	at folk.sisby.tinkerers_smithing.data.SmithingMaterialLoader.apply(SmithingMaterialLoader.java:55) ~[transformed-mod-tinkerers-smithing.i0:0/:?]
	at folk.sisby.tinkerers_smithing.data.SmithingMaterialLoader.method_18788(SmithingMaterialLoader.java:18) ~[transformed-mod-tinkerers-smithing.i0:0/:?]
```

Full client log: 
[latest.log](https://github.com/sisby-folk/tinkerers-smithing/files/12329548/latest.log)

**Comment by sisby-folk** [comment]
oh classic - I definitely know why that is. you're on quilt loader so just use the previous version for now.

**Comment by sisby-folk** [comment]
fixed in 2.3.1
